launchable/commands/subset.py

```diff
@@ -526,88 +534,112 @@ def get_payload(
 
             return payload
 
+        def _collect_potential_test_files(self):
+            LOOSE_TEST_FILE_PATTERN = r'(\.(test|spec)\.|_test\.|Test\.|Spec\.|test/|tests/|__tests__/|src/test/)'
+            EXCLUDE_PATTERN = r'\.(xml|json|txt|yml|yaml|md)$'
+
+            git_managed_files = []
+            try:
+                git_managed_files = subprocess.run(['git', 'ls-files'], stdout=subprocess.PIPE,
```

Comment:
`subprocess.CalledProcessError` doc says the `run` method needs to be invoked with `check=True` for the exception to be raised.